website/src/components/Hero/Hero.jsx

```diff
@@ -11,7 +11,13 @@ const Hero = ({ className, children, heading, description, ...rest }) => {
     <section className={classes} {...rest}>
       {heading !== '' ? <Heading level={1}>{heading}</Heading> : ''}
 
-      {description !== '' ? <Heading level={4}>{description}</Heading> : ''}
+      {description !== '' ? (
+        <Heading level={4} element="h2">
+          {description}
+        </Heading>
+      ) : (
+        ''
+      )}
```

Comment:
This notation _might_ work? (it's widely used in the React world as it's shorter -- not a blocker) 

```suggestion
      {description && (
        <Heading level={4} element="h2">
          {description}
        </Heading>
      )}
```

Reply:
thanks!